src/main/kotlin/lotto/LottoNumber.kt

```diff
@@ -0,0 +1,14 @@
+package lotto
+
+class LottoNumber private constructor(val number: Int) {
+    companion object {
+        const val MIN = 1
+        const val MAX = 45
+        private val NUMBERS: Map<Int, LottoNumber>
```

Comment:
You’ve implemented a strategy to limit unbounded instance creation - that’s a great approach! 😊